File: BriefAssistant/Models/InitialBriefInfo.cs
using System;

namespace BriefAssistant.Models
{
    public class InitialBriefInfo
    {
        public Guid Id { get; set; }
        public string IssuesPresented { get; set; }
        public string OralArgumentStatement { get; set; }
        public string PublicationStatement { get; set; }
        public string CaseFactsStatement { get; set; }
        public string AppendixDocuments { get; set; }

        public BriefInfo BriefInfo { get; set; }
    }
}

File: BriefAssistant/ClientApp/src/app/models/InitialBriefInfo.ts
import { BriefInfo } from "./BriefInfo";

export class InitialBriefInfo {
  id: string;
  briefInfo: BriefInfo;
  appendixDocuments: string;
  caseFactsStatement: string;
  oralArgumentStatement: string;
  publicationStatement: string;
  issuesPresented: string;

  constructor() {
    this.briefInfo = new BriefInfo();
    this.appendixDocuments = "";
    this.caseFactsStatement = "";
    this.oralArgumentStatement = "";
    this.publicationStatement = "";
    this.issuesPresented = "";
	}
}
